elastic-search-support/src/main/resources/application.yml

```diff
@@ -16,7 +16,7 @@ elasticsearch:
     defaultBoolean: '{"type" : "boolean"}'
     defaultLong: '{"type" : "long"}'
     defaultDouble: '{"type" : "double"}'
-    ccdPhoneUK: '{"type" : "text", "analyzer": "phone_number_analyzer"}'
+    ccdPhoneUK: '{"type" : "text", "fields": { "keyword": { "type": "keyword", "ignore_above": 256, "normalizer": "lowercase_normalizer"}}, "analyzer": "phone_number_analyzer"}'
     ccdDocument: '{
                     "properties":{
```

Comment:
ccdPhoneUK has been also fixed in this ticket

Reply:
Thanks 👍